src/openforms/emails/tests/test_tasks.py

```diff
@@ -15,6 +15,7 @@
 from ..tasks import send_email_digest
 
 
+@override_settings(LANGUAGE_CODE="en")
```

Comment:
:(

I explicitly mentioned this on Slack as not being a sustainable approach. I'm only merging this now to not further block the release.